### PR TITLE
36: Dark mode support toggle

### PR DESCRIPTION
**Pull request by shakedlv**

### CODE REVIEW

**Review by shakedlv — COMMENTED** [review]
To - Do :
1) Add all colors  from figma with a proper naming convention.
2) Move theme switcher toggle to a better position

File: src/components/theme-switcher/theme-switcher.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
+@import '../../index.css';
```
Comment (ofir-zeitoun) [inline src/components/theme-switcher/theme-switcher.css:1]: try move import index.css to main

File: src/components/theme-switcher/theme-switcher.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    font-weight: 500;
+}
+
+.btn-color-mode-switch{
+    display: inline-block;
+    margin: 0px;
+    position: relative;
+}
+
+.btn-color-mode-switch > label.btn-color-mode-switch-inner{
```
Comment (ofir-zeitoun) [inline src/components/theme-switcher/theme-switcher.css:20]: you can nest it in lines 14-18

File: src/components/theme-switcher/theme-switcher.css [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    margin: 0px;
```
Comment (ofir-zeitoun) [inline src/components/theme-switcher/theme-switcher.css:21]: change px to rem

**Review by ofir-zeitoun — CHANGES_REQUESTED** [review]
minor fixes

File: src/components/theme-switcher/theme-switcher.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react';
+import './theme-switcher.css';
+
+type ThemeMode = "dark" | "light";
+
+export const ThemeSwitcher = () => {
+  const isSystemDark = window.matchMedia('(prefers-color-scheme: dark)').matches;
```
Comment (ofir-zeitoun) [inline src/components/theme-switcher/theme-switcher.tsx:7]: use useThemeMode hook

**Review by shakedlv — COMMENTED** [review]
Fixed Comments
- Moved css imports to main
- nested css 
- formated css , change to rem
- export theme logic to useTheme hook

File: src/components/theme-switcher/theme-switcher.css (hunk 1 above)
Comment (ofir-zeitoun) [reply src/components/theme-switcher/theme-switcher.css]: Couldn't move?

File: src/components/nav-bar/nav-bar.tsx [hunk 5]
```diff
@@ -8,6 +8,7 @@ import {
 import { useAuth0 } from "@auth0/auth0-react";
 import { ConnectionHealth } from "../connection-health";
 import "./nav-bar.css";
+import { ThemeSwitcher } from "../theme-switcher";
```
Comment (ofir-zeitoun) [inline src/components/nav-bar/nav-bar.tsx:11]: move up

**Review by ofir-zeitoun — APPROVED** [review]

**Pull request merged**